scripts/app.js

```diff
@@ -1,39 +1,55 @@
-var youtube_search = 'https://www.googleapis.com/youtube/v3/search';
 
-function getDataFromApi(searchTerm, callback) {
-  var query = {
+// This is the YouTube API call Object. It also has a prototypical inheritance of the 'callAPI' method
+var Data = function(searchTerm, callback){
+  this.url = 'https://www.googleapis.com/youtube/v3/search';
+  this.query = {
     part: 'snippet',
-    q: searchTerm,    
+    q: searchTerm,
     key: 'AIzaSyBN_2IeT7Ez2Wp2gsecionVgTUc4uL-hh4',
   }
-  $.getJSON(youtube_search, query, callback);
+  console.log(typeof(Data.callAPI));
 }
 
 
+// The API call to YouTube
+Data.prototype.callAPI = function(){
+  $.getJSON(url, query, callback);
+}
+
+// Captures data from Youtube API and has a prototypical inheritance of the 'render' method
+var Items = function(data){
+  this.col = $('<div>').addClass('col-4');
+  this.box = $('<div>').addClass('box');
+  this.desc = $('<div>').addClass('description');
+  this.href = $('<a>').attr('href', 'https://www.youtube.com/watch?v=' + data.id.videoId).attr('target', '_blank');
+  this.image = $('<img>').attr('src', data.snippet.thumbnails.medium.url);
+  this.title = $('<p>').text(data.snippet.description);
+  this.remove = $('<button>').attr('input', 'button').attr('class','remove').text('Delete');
+
+   
+}
+
+// This renders method that is inherited by Items object
+Items.prototype.render = function(){
+  var titl = $(desc).append(title).append('<br>').append(remove)
+        
+
+  var boxy = $(box).append(image).append(titl);
+  var link = $(href).append(boxy);
+        
+
+  var col4 = $(col).append(link);
+  $('.js-search-results').append(col4);
+};
+
+
+
 function displaySearchData(data) {
-    var result = '';
-    console.log(data)
    
     if (data.items.length !== 0) {
      	data.items.forEach(function(data) {
-     			var col = $('<div>').addClass('col-4');
-     			var box = $('<div>').addClass('box');
-     			var desc = $('<div>').addClass('description');
-     			var href = $('<a>').attr('href', 'https://www.youtube.com/watch?v=' + data.id.videoId).attr('target', '_blank');
-     			var image = $('<img>').attr('src', data.snippet.thumbnails.medium.url);
-     			var title = $('<p>').text(data.snippet.description);
-     			var remove = $('<button>').attr('input', 'button').attr('class','remove').text('Delete');
-
-     			var titl = $(desc).append(title).append('<br>').append(remove);
-     			
-
-     			var boxy = $(box).append(image).append(titl);
-     			var link = $(href).append(boxy);
-     			
-
-     			//var boxy = $(box).append(href).append(titl)
-     			var col4 = $(col).append(link);
-     			$('.js-search-results').append(col4);
+        var item = Items();
```

Comment:
I think you're going to want to pass the `data` object into the `Items` constructor function as that's how it's declared. And then you can call `item.render()` without passing `data` as an argument.